napalm_logs/utils/__init__.py

```diff
@@ -174,6 +174,38 @@ def stop(self):
         self.ssl_skt.close()
 
 
+class Replace:
```

Comment:
I have added as functions, but prefixed them with `replace_` so there can be no conflicts between other function.